src/google_apps_clj/credentials.clj

```diff
@@ -87,10 +89,41 @@
       (.setRefreshToken (:refresh-token auth-map))
       (.setTokenType (:token-type auth-map)))))
 
-(t/ann ^:no-check build-credential [GoogleCtx -> HttpRequestInitializer])
+(t/ann credential-with-scopes [GoogleCredential (t/Seq t/Str) -> GoogleCredential])
```

Comment:
It would be boss if we could use a stronger type than t/Str here. Do we know if there is a discrete set of scopes knowable at compile-time?